STYLE.md

```diff
@@ -85,29 +85,40 @@ For links to other pages or the web, use standard Markdown instead of JSX or HTM
 [Meshes](../parts/meshes.md)
 ```
 
-To link to the Roblox Engine API, use the following syntax. The parser detects these keywords and creates monospaced links to the API page:
+To link to a page in the Roblox Engine API, use the following syntax. The parser detects these keywords and creates monospaced links to the API page:
 
 - `` `Class.Name` ``
 - `` `Class.Name.Property` ``
 - `` `Class.Name:Method()` ``
+- `` `Class.Name:Method(argument)` ``
+- `` `Class.Name:Method(arg1, arg2, arg3, ...)` ``
 - `` `Class.Name.Event` ``
 - `` `Class.Name.Callback` ``
 
 - `` `Datatype.Name` ``
 - `` `Datatype.Name:Method()` ``
+- `` `Datatype.Name:Method(argument)` ``
+- `` `Datatype.Name:Method(arg1, arg2, arg3, ...)` ``
 - `` `Datatype.Name.constructor()` ``
+- `` `Datatype.Name.constructor(argument)` ``
+- `` `Datatype.Name.constructor(arg1, arg2, arg3, ...)` ``
 
 - `` `Enum.Name` ``
 
 - `` `Global.LuaGlobals.Name` ``
 - `` `Global.RobloxGlobals.Name` ``
 - `` `Library.Name` ``
 - `` `Library.Name.function()` ``
+- `` `Library.Name.function(argument)` ``
+- `` `Library.Name.function(arg1, arg2, arg3, ...)` ``
+
+The default link text will display without the initial API type prefix (Class, Datatype, Enum, Global, Library).
```

Comment:
```suggestion
The default link text will display without the initial API type prefix (`Class.`, `Datatype.`, `Enum.`, `Global.`, `Library.`). For example, `` `Class.BasePart` `` will render as simply `Class.BasePart`.
```

Reply:
I think this will render out properly once viewed? Need to confirm though, that my usage of backticks will do what I want it to...